src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -6,6 +6,7 @@ Release History
 0.1.2 (2017-04-03)
 ++++++++++++++++++++
 
+* appservice: adding ftp publishing url to create webapp and show webapp
```

Comment:
Can you create a new heading with title 'unreleased' and include this in there as it will be part of the next release, not the one that just happened today.